### PR TITLE
Check enrichment numeric

### PR DESCRIPTION
**Pull request by curadomr**
Provides checks to test if 'nominal' and 'adjusted' columns for Enrichment tables are numeric. Also includes checks in tinytest file (testCheck.R)

### CODE REVIEW

**Review by jdblischak — CHANGES_REQUESTED** [review]
Code looks good and the tests pass. Is the plan to [create a new release](https://github.com/abbvie-external/OmicNavigator/blob/main/CONTRIBUTING.md#tag-a-new-release) immediately? Or can this wait until the next release of the web app (which will automatically create a new R package release)?

To do:

* Bump the version. For immediate release, bump to 1.13.10. Otherwise use a dev version, 1.13.9.1
* Add a bullet to `NEWS.md`. For immediate release, also manually add the header `# 1.13.10`. Otherwise don't include a header

**Comment by bengalengel** [comment]
let's do an immediate release please!

**Review by jdblischak — APPROVED** [review]

**Comment by jdblischak** [comment]
I can't merge since I don't have write access. For an immediate release, I recommend the following steps:

1. Squash and merge this PR (ie combine all the commits into a single commit with an informative message)
2. In your terminal, pull the latest changes `git pull upstream main`
3. Follow the remaining instructions to [tag a new release](https://github.com/abbvie-external/OmicNavigator/blob/main/CONTRIBUTING.md#tag-a-new-release), starting with the step to `git tag -a v1.13.10 -m "v1.13.10"`

**Pull request merged**
